frontend/context/Web3AuthProvider.tsx

```diff
@@ -0,0 +1,33 @@
+import { WEB3AUTH_NETWORK } from '@web3auth/modal';
+import {
+  type Web3AuthContextConfig,
+  Web3AuthProvider as Web3AuthPackageProvider,
+} from '@web3auth/modal/react';
+import { PropsWithChildren } from 'react';
+
+const clientId = `${process.env.WEB3AUTH_CLIENT_ID}`;
+
+const web3AuthContextConfig: Web3AuthContextConfig = {
+  web3AuthOptions: {
+    clientId,
+    web3AuthNetwork: WEB3AUTH_NETWORK.SAPPHIRE_DEVNET,
+    mfaLevel: 'optional',
+    mfaSettings: {
+      socialBackupFactor: { enable: true, mandatory: true },
+      passkeysFactor: { enable: true, mandatory: false },
+      authenticatorFactor: { enable: true, mandatory: false },
```

Comment:
Better to explicitly set up all the options, even if enabled: false. Also missing priority